src/utils/sformat.fs

```diff
@@ -227,20 +228,21 @@ namespace Microsoft.FSharp.Text.StructuredFormat
             "uint64"
             "unativeint"
           ] |> Set.ofList
-        let tagClass name = if Set.contains name keywordTypes then TaggedText.Keyword name else TaggedText.Class name
-        let tagUnionCase = TaggedText.UnionCase
-        let tagDelegate = TaggedText.Delegate
-        let tagEnum = TaggedText.Enum
-        let tagEvent = TaggedText.Event
+
+        let tagClass name = if Set.contains name keywordTypes then TaggedText.Keyword name else TaggedText.Class(None, name)
+        let tagUnionCase = blank >> TaggedText.UnionCase
```

Comment:
If you want to avoid the parentheses in the signature file then just use an explicit argument rather than ``>>``